### PR TITLE
fix&update fusion material

### PR DESCRIPTION
**Pull request by nekrozar**
https://github.com/Fluorohydride/ygopro-core/pull/242

### CODE REVIEW

**Comment by mercury233** [comment]
got this:
[Script Error]: [string "./script/utility.lua"]:1072: Parameter 1 should be "Group".
replay:
[444play.zip](https://github.com/Fluorohydride/ygopro-scripts/files/3817441/444play.zip)

**Comment by nekrozar** [comment]
Sorry, I can't play this replay.
I would like to get more information.

**Comment by mercury233** [comment]
i didn't manage to reproduce this issue before i got this replay from another user. let me try again.

**Comment by mercury233** [comment]
OK, I guess I understand it:
1. Place _Trishula, the Dragon of Icy Imprisonment_ and _Dragonmaid Husky_ in extra deck
2. Summon a Dragonmaid to field, discard a Dragonmaid to grave
3. Activate _Dragon's Mirror_
4. This error show up.

It is because when it check
https://github.com/Fluorohydride/ygopro-scripts/blob/master/c71490127.lua#L31
the `mg1` is altered by
https://github.com/Fluorohydride/ygopro-core/pull/242/files#diff-8e1e3342c6da0ff99bc2f53f27026b12R2804
when Trishula is being checked

**Comment by nekrozar** [comment]
I could confirm these problems.
https://github.com/Fluorohydride/ygopro-core/commit/e78e83af0b38be35879dca69720a3bbfc055a0f4 fixed. Thank you.
